docs/contracts/wsteth.md

```diff
@@ -3,25 +3,45 @@
 - [Source Code](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.6.12/WstETH.sol)
 - [Deployed Contract](https://etherscan.io/token/0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0)
 
-It's an ERC20 token that represents the account's share of the total
-supply of stETH tokens. WstETH token's balance only changes on transfers,
-unlike StETH that is also changed when oracles report staking rewards and
-penalties. It's a "power user" token for DeFi protocols which don't
-support rebasable tokens.
+## What is wrapped stETH (wstETH)?
 
-The contract is also a trustless wrapper that accepts stETH tokens and mints
-wstETH in return. Then the user unwraps, the contract burns user's wstETH
-and sends user locked stETH in return.
+It's an [ERC-20](https://eips.ethereum.org/EIPS/eip-20) value-accruing token
+wrapper for `stETH`. Its balance does not change with each oracle report, but its
+value in `stETH` does. Internally, it represents the user's share of the total
+supply of `stETH` tokens.
```

Comment:
may refer to https://docs.lido.fi/guides/steth-integration-guide#steth-internals-share-mechanics or https://docs.lido.fi/contracts/lido#rebase